Formula/ark.rb

```diff
@@ -0,0 +1,25 @@
+class Ark < Formula
+  desc "Disaster recovery for Kubernetes cluster resources and persistent volumes"
+  homepage "https://github.com/heptio/ark"
+  url "https://github.com/heptio/ark/archive/v0.8.2.tar.gz"
+  sha256 "149b5262e9e113a8963cd97e1b32c6cfc1eea23b2cc45ffdda9b81a789da3e26"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = MacOS.prefer_64_bit? ? "amd64" : "x86"
+    contents = Dir["{*,.git,.gitignore}"]
+    (buildpath/"src/github.com/heptio/ark").install contents
+
+    cd "src/github.com/heptio/ark" do
+      system "go", "build", "-o", "_output/ark", "-installsuffix", "static", "cmd/ark/main.go"
+      bin.install "_output/ark"
+    end
+  end
+
+  test do
+    assert_match "Heptio Ark is a tool for managing disaster recovery, specifically for Kubernetes", shell_output("#{bin}/ark 2>&1")
```

Comment:
The test needs to exercise and verify the functionality of the software.

Reply:
I added two test cases for the version output (and also found a missing build flag through this) and for setting and getting the client configuration.
Further functionality of the cli client would need a working Kubernetes cluster.